rules/Proj0001.md

```diff
@@ -9,4 +9,4 @@ Therefore all rules in this package try to resolve the project file based on
 (naming and location) conventions.
 
 This rule reports when the project file could not be located, due to whatever
-reason.
+reasonz.
```

Comment:
Tot test something I suppose?

Reply:
Yes, this github action runs LanguageTool which is supposed to do some spelling and grammar checks. I'm trying to trigger it, but with little success so far.

Reply:
@CptWesley You try to find a bot that is as good as @LauraXiulan ?

Reply:
Basically 😊

Reply:
Apperently the spell checker is disabled by default...